app/components/user_list_row/index.tsx

```diff
@@ -55,12 +56,13 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
             flexDirection: 'column',
             flex: 1,
         },
-        displayName: {
-            fontSize: 15,
+        username: {
+            fontSize: 12,
             color: changeOpacity(theme.centerChannelColor, 0.5),
         },
-        username: {
-            fontSize: 15,
+        displayName: {
+            fontSize: 16,
```

Comment:
We should use the typography helper here. It's not using right font right now

`...typography('Body', 200, 'regular'),`